src/interpreter.rs

```diff
@@ -276,7 +276,10 @@ impl Interpreter {
                         ));
                     }
 
-                    return func.call(self, params);
+                    return match func.call(self, params){
+                        Ok(v) => Ok(v),
+                        Err(e) => Err(Error::new(tok.lineinfo, e.0, e.1)),
+                    };
```

Comment:
I don't think this should be changed. This is a hack - we need proper tracebacks on errors by this point.

Reply:
I know, I just add it there so that it actually gives off the correct lineinfo value. Which is it's a temporary fix

Reply:
We don't want temporary fixes, this code is going to be effectively permanent. If tracebacks are required, they should probably be implemented now before this is merged. Another issue is that this means errors inside user functions will not have their true source reported.